packages/date-picker/package.json

```diff
@@ -38,6 +38,8 @@
     "classnames": "^2.3.1",
     "react-day-picker": "^7.4.9",
     "react-focus-lock": "^2.8.1",
+    "react-focus-on": "^3.5.4",
+    "aria-hidden": "^1.1.3",
```

Comment:
Now we're using `react-focus-on` I don't think we need `aria-hidden`

Reply:
yep will remove

Reply:
and focus lock